modules/rds/main.tf

```diff
@@ -0,0 +1,35 @@
+module "rds" {
+  source  = "terraform-aws-modules/rds-aurora/aws"
+  version = "9.0.0"
+
+  name = "${var.namespace}-aurora-db"
+
+  engine         = "aurora-postgresql"
+  engine_version = 15.3
+
+  vpc_id                 = var.vpc_id
+  subnets                = var.subnets
+  vpc_security_group_ids = var.vpc_security_group_ids
+
+  instance_class = var.instance_class
+  instances = {
+    one = {}
+  }
```

Comment:
`instances` is optional and would be the same as the default right? Do we need this? 💭

Reply:
it's not like the default `{}` FYI.
We need this to state that we have an instance with `instance_class` from above